gcore/src/prog.rs

```diff
@@ -71,6 +81,87 @@ mod sys {
 ///     let submitted_code: CodeHash =
 ///         hex_literal::hex!("abf3746e72a6e8740bd9e12b879fbdd59e052cb390f116454e9116c22021ae4a")
 ///             .into();
+///     let new_program_id = prog::create_program(submitted_code, &get().to_le_bytes(), b"", 0);
```

Comment:
I would like to ref any `*with_gas` to their gasless variant, but with explanation what gas limit specifying changes.

Reply:
And not only for this method, for other sendings too.

Reply:
> How do you think about my suggestion, @shamilsan?

Completely agree.